src/System.IO.FileSystem/tests/Directory/CreateDirectory.cs

```diff
@@ -127,7 +127,7 @@ public void ValidPathWithTrailingSlash()
         }
 
         [ConditionalFact(nameof(UsingNewNormalization))]
-        [SkipOnTargetFramework(Tfm.BelowNet462 | Tfm.Core50, "long path support added in 4.6.2")]
+        [SkipOnTargetFramework(Tfm.Core50)]
```

Comment:
public const TargetFrameworkMonikers Core50 = TargetFrameworkMonikers.Netcore50 | TargetFrameworkMonikers.Netcore50aot; which is mainly the older version of the UAP and UAPAOT. 

@safern we decided to test against the latest version only of UAP. so we need to mark the test with UAP/aot if it is failing there or just remove the attribute it if it not failing.

CC @joperezr

---
In reply to: [104301915](https://github.com/dotnet/corefx/pull/16706#discussion_r104301915) [](ancestors = 104301915)